### PR TITLE
Move & restructure Networks page

### PR DESCRIPTION
**Pull request by mishmosh**
- Moves Networks info out of Build section & to project level, so that Miners can also find it
- Updates sidebar & homepage links
- Adds top-level table to compare networks.
- Reorganizes each network's details with tables (feedback especially welcome here).

Questions for reviewers
- A main goal of this PR is to improve the overview of all networks, and make it very easy for people to pick the right one for them. Is that achieved? Any improvements to suggest?
- Are each network's details very clear & findable? Any improvements to suggest?
- Do the general "connection instructions" work? Another option is to revert to individual instructions (with branch/tag name pasted in for each), but I partly wanted to reinforce that all the steps for joining any network is basically the same.

Partially addresses #174. This PR does not remove redundant information in `start-building/interacting-with-the-network.md`, which will come in a follow-up PR.

### CODE REVIEW

**Comment by mishmosh** [comment]
@terichadbourne do you see any obvious reasons why tests are failing? I double-checked the likeliest candidates (`config.js` and `Home.vue`) but might be missing something.

**Comment by terichadbourne** [comment]
@mishmosh I see this in the failing builds which suggests there's a syntax error somewhere in the config file. 

image

Looking at the code you changed I don't see where you would have messed up the syntax. Only thought I had from looking at it was that maybe the sidebarDepth in the metadata of the article itself if either wrong or unneeded, but I had this thought before finding the log pasted above.

**Comment by terichadbourne** [comment]
I would have assumed the last `}` in there was meant to close `module.exports` but it actually closes on line 253 right above the error. I'm not familiar enough with this file to know at first glance what's correctly v incorrectly nested fields and what's not tabbed properly, which would help find the messed up syntax. My guess is it would be more immediately apparent to @cwaring who's spent more time staring at this config file. :) 

I still can't see how your recent changes could have messed up the syntax, though. Any chance you started from a branch that was already bad?

File: docs/how-to/networks.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+------
+
+## Devnets
+
+Several developer networks (also called devnets) are available with various configurations and performance characteristics. The recommended networks for most users are [calibration](#calibration-devnet) (for miners) and [nerpa](#nerpa-devnet) (for developers). 
+
+### Calibration Devnet
+
+Calibration is an up-to-date devnet, best for miners preparing for the [Space Race]() mining competition. Miners can practice in real competition conditions and see how their performance influences hypothetical competition standings.
```
Comment (terichadbourne) [inline docs/how-to/networks.md:66]:
```suggestion
Calibration is an up-to-date devnet, best for miners preparing for the [Space Race](https://filecoin.io/blog/getting-ready-testnet-incentives/) mining competition. Miners can practice in real competition conditions and see how their performance influences hypothetical competition standings. 
```

**Comment by terichadbourne** [comment]
After trying merging master into your branch locally to see if there was a previous error, I got a new error (🎉)about a missing link.  Nice testing work by @cwaring! 

@mishmosh I used a link you had used elsewhere for Space Race, so you should double check it's what you intended to link to. 

I have no idea why merging master into networks (which doesn't show any actual changes in the commit) made this work, but the Netlify tests show as passing now, so my job here is done. ¯\_(ツ)_/¯

**Comment by mishmosh** [comment]
Very odd. Thank you for exorcising the ghosts @terichadbourne @cwaring.

**Pull request merged**